dashboard/test/controllers/api/v1/section_instructors_controller_test.rb

```diff
@@ -0,0 +1,161 @@
+require 'test_helper'
```

Comment:
Should we have a test for not allowing a 7th instructor?

Reply:
Done.